igcollect/ping.py

```diff
@@ -113,24 +144,35 @@ def pings(hosts, count, timeout, delay):
 
 
 def print_ping(prefix, data, timestamp):
+    """ prints the data in a format we can collect
+
+        Parameters:
+            prefix (str): were the script will put the data in graphite
+            data (dict(floar)): the data gathered (pings, min, may, avg, std)
+            timestamp (str): the timestamp as a number
+
+    """
+
+    pings = data['pings']
+
     template = (
-        str(prefix) + '.' + data['dest'] + '.{} {} ' + timestamp
+        prefix + '.' + data['dest'] + '.{} {} ' + timestamp
     )
 
-    for num, ping in enumerate(data['pings'], 1):
+    for num, ping in enumerate(pings, 1):
         print(template.format('ping' + str(num), ping))
 
     print(template.format('fails', data['fails']))
 
-    if len(data['pings']) == 0:
-        raise Exception('something went horrible wrong: check your fping')
+    if len(pings):
+        # If the length is 0 we ignore the entries
```

Comment:
What is the reason to ignore all entries? The other entries are possible to print.

Reply:
we just ignore the entries of the not send stuff every other thing will be written